test/integ_tests/conftest.py

```diff
@@ -171,29 +171,21 @@ def job_failed_name(request):
 
 @pytest.fixture(scope="session", autouse=True)
 def completed_quantum_job(job_completed_name):
-    job_arn = (
-        [
-            job["jobArn"]
-            for job in boto3.client("braket").search_jobs(filters=[])["jobs"]
-            if job["jobName"] == job_completed_name
-        ][0]
-        if os.getenv("JOBS_STARTED")
-        else None
-    )
-
-    return AwsQuantumJob(arn=job_arn) if os.getenv("JOBS_STARTED") else None
+    job_arn = [
```

Comment:
Can you run the integ tests against these locally once? I want to make sure if the environment variable is not used elsewhere.

Reply:
The env var is used elsewhere but that is on the distribution worker

Reply:
Discussed offline; this will be a post launch fix.